### DISCUSSION TITLE
August Challenge

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#167] [Monthly Challenges]
@Virtual-Coffee/monthly-challenge let's talk ideas for next month!

We've had two solid months of participation. I think if we're looking at @tkshill's[ structure,](https://github.com/Virtual-Coffee/VC-Contributors/discussions/144#discussioncomment-782718) we're due for a recover/caring month.

We haven't implemented this structure yet, so we don't have to do it if you have other ideas, but I'd love to hear ideas for this too.

**Comment by tkshill** [comment]
I agree that considering Hacktoberfest season is happening soon, I would love August to be a month focused on recovery. Maybe the "challenge" for the month can be focused around some healthy habits. Sleep, rest, tech debt reduction, but like, activities that dont involve large amounts of "new" work or intense organization on our parts.

**Reply by aurelieverrot** [reply]
Self-care and code-care?

Example: It can be healthy habits for healthy devs. These habits can be mind and body centered (drink, move, read, meditate, rearrange your work station.....) or code centered (review your ReadMe, clean your code, refresh your GitHub repo....) or both.

**Reply by BekahHW** [reply]
I LOVE this!

**Reply by ClJarvis** [reply]
It might be strange but it is calming to clean a work station.  I'll be taking time to do that this week.  Some thing to promote rest would be great.  Maybe a list on how to snooze certain apps.

**Reply by LSanchez17** [reply]
I like this as well. Should we have like a weekly list of self care/code care tips that we can share?

**Reply by dominicduffin1** [reply]
I also like this!

**Comment by vanessacor** [comment]
Healthy habits for healthy devs!! Love this :D

Yes Chris it does, for me it certain helps my mental state. After cleaning
my work station I feel more energize and more focus.

I think being also code centred it fits really well as a preparation for
Hacktoberfest
